systers_portal/templates/meetup/base.html

```diff
@@ -1,14 +1,14 @@
 {% extends "base.html" %}
 
 {% block title %}
-  {{ object.name }}
+  {{ meetup_location.name|capfirst }}
```

Comment:
If an admin decided to name Meetup Location with a small letter, then she knows what she's doing. If we do capfirst, then we need to do capfirst everywhere. If we miss one place (and since this is not documented, future developers might not know about this), then it becomes inconsistent. If the admin made a mistake, then she will she it straight away (if we don't use capfirst). Otherwise she might never know she has created a meetup location with a small letter. If you want to enforce this kind of sanitizing, it should be done at the ORM level.